.github/workflows/test_python.yaml

```diff
@@ -31,43 +38,92 @@ jobs:
+          # choco install doxygen.install --version 1.9.6
+          #
+          # https://github.com/actions/runner-images/issues/92#issuecomment-1172456433
+          $url = "https://www.doxygen.nl/files/doxygen-1.9.6.windows.x64.bin.zip"
+          Start-DownloadWithRetry -Url $url -Name "doxygen.zip" -DownloadPath "C:\"
+          Extract-7Zip -Path  "C:\doxygen.zip" -DestinationPath "C:\Program Files\doxygen"
+          # Extracts to C:\Program Files\doxygen
+          #   - doxygen.exe
+          #   - doxyindexer.exe
+          #   - doxysearch.cgi.exe
+          #   - doxywizard.exe
+          #   - libclang.dll
+          echo "C:\Program Files\doxygen" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      ##################################################################################
       - name: Doxygen Version Dump
         run: doxygen --version
-      ##########################################################################
+      ##################################################################################
       - name: Use Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install Python Tools
         run: |
-          pip install -U tox codecov coverage
+          pip install -U pip tox codecov coverage
+      # Sphinx 3.0.0 ###################################################################
       - name: Test Python ${{ matrix.python-version }} / sphinx==3.0.0
         env:
-          # NOTE: in the future, we may want to use a larger matrix with more
-          # combinations of sphinx and breathe, but for now it is not necessary.
+          # NOTE: this is the current minimum supported.
           SPHINX_VERSION: '==3.0.0'
         run: |
           tox -e py -- --cov-report xml:coverage.xml --cov
       - name: Upload Code Coverage for Python ${{ matrix.python-version }} / sphinx==3.0.0
         run: |
           codecov --required -X gcov -f coverage.xml --name "[GHA] ${{ matrix.os }}-py${{ matrix.python-version }}-sphinx3.0.0"
       # Don't let code coverage utilities share anything, force clean it all.
-      - name: Cleanup Python ${{ matrix.python-version }} / sphinx==3.0.0 Artifacts
+      - name: Cleanup Python ${{ matrix.python-version }} Testing / Coverage Artifacts
         run: |
           mv .gitignore nolongerignored
           git clean -n
           git clean -f
           git reset --hard
+      # Sphinx 4.x #####################################################################
       - name: Test Python ${{ matrix.python-version }} / sphinx 4.x
         env:
-          # NOTE: in the future, we may want to use a larger matrix with more
-          # combinations of sphinx and breathe, but for now it is not necessary.
-          SPHINX_VERSION: '>=4<5'
+          # Test sphinx 4.x.
+          SPHINX_VERSION: '>=4,<5'
```

Comment:
see revised `test_python.yaml` if interested in why `SPHINX_VERSION` was working like this.  Want to test 4.x, 5.x, and 6.x.  That code was remnant from when 4.x was latest version